tests/Common/TracerTestTrait.php

```diff
@@ -182,7 +182,19 @@ public function inCli($scriptPath, $customEnvs = [], $customInis = [], $argument
         return $this->parseTracesFromDumpedData();
     }
 
-    public function executeCli($scriptPath, $customEnvs = [], $customInis = [], $arguments = '')
+    /**
+     * This method executes a single script with the provided configuration.
+     */
+    public function inCliWithOutput($scriptPath, $customEnvs = [], $customInis = [], $arguments = '')
```

Comment:
[Non blocking] Since there are only 5 usages of the original `inCli` I think we could instead:
1. invert the return order from this method to `[$this->parseTracesFromDumpedData(), $this->executeCli($scriptPath, $customEnvs, $customInis, $arguments, true)]`;
2. replace the five usages of `$traces = ...->inCli(...);` with `list($traces) = ...->inCli(...);`

I think some less code to maintain, what do you think?

Reply:
Done, feel free to re-ack and possibly consider inclusion in 0.69.0 as well?

Reply:
I'd say that 0.69.0 can be released without this, since it would mean postposing one more day for a use case that is not common. 0.70.0 will be out shortly in any case.